.circleci/config.yml

```diff
@@ -65,11 +75,16 @@ jobs:
       - image: circleci/openjdk:8-jdk
     steps:
       - checkout
+      - run:
+          name: Generate cache key
+          command: |
+            cd languages/java/app-encryption
+            ../../../scripts/checksum.sh /tmp/checksum.txt pom.xml
```

Comment:
1) use `$CIRCLE_WORKING_DIRECTORY` instead of the relative path
2) nit: now that i think of it, wonder if `.circleci` directory may be even more appropriate than `scripts` (since it is purely for circleci at this point)?

Reply:
discussed offline: we'll come back to this one, hence the TODO/note